clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/auth/DockerBearerToken.groovy

```diff
@@ -20,9 +20,9 @@ import groovy.transform.ToString
 
 @ToString(includeNames = true)
 class DockerBearerToken {
-  // One of token or bearer_token will be filled by the request.
+  // One of token or access_token will be filled by the request.
   String token
-  String bearer_token
+  String access_token
```

Comment:
While I'm not positive, there's a chance that `bearer_token` is used by some system (albeit incorrectly). Could you keep it in, and read it only when `token` & `access_token` are empty?

Reply:
Certainly.

Reply:
Thanks :)

Reply:
Sorry, not particularly well versed in Groovy's Elvis operators, does the change make sense?